### PR TITLE
refactor: switch to electron-builder

### LINKED ISSUE #106: [BUG] Installer eating up memory
**Installer Version**
latest from 15.1.2021

**Describe the bug**

Even when the installer is closed it remains active and eats up an incredible amount memory (>25GB).

**To Reproduce**

1. Run installer
2. Close it again
3. Watch memory usage increase over time
image

**Expected behavior**

close the application when exiting

**Additional context**

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by Benjozork**
Fixes #106

## Summary of Changes

* Switch to `electron-builder`
* Switch to `NSIS` installer and auto-updater (fixes #106)
* Switch to in-house webpack config (instead of automagically generated one)

## Known issues:
* Webpack does not update automatically if codebase is changed. You will need to re-run `npm run start`

Discord username (if different from GitHub): someperson#4953

### CODE REVIEW

File: .github/workflows/release.yaml
Comment (ZigTag) [inline .github/workflows/release.yaml:28]: ./dist, not /dist afaik

File: .github/workflows/v2-dev.yaml
Comment (ZigTag) [inline .github/workflows/v2-dev.yaml:26]: Here as well

**Review by ZigTag — CHANGES_REQUESTED** [review]

File: package.json
Comment (ZigTag) [inline package.json:4]: Make sure to change this before merging

**Review by ZigTag — APPROVED** [review]
LGTM, just change version name

File: package.json
Comment (FoxtrotSierra6829) [reply package.json:4]: Nah that's fine, merge doesn't mean it gets released in that state. I'll create a separate branch for the actual release

Comment (ZigTag) [reply package.json:4]: Discussed in discord, resolving

**Review by ZigTag — APPROVED** [review]
LGTM

File: .github/CHANGELOG.md
Comment (Benjozork) [inline .github/CHANGELOG.md:14]:
```suggestion
1. [LOGIC] Installation method changed from Electron-Forge/Squirel to electron-builder/NSIS to prevent future memory leaks during update @Benjozork - @FoxtrotSierra6829
1. [LOGIC] Remove converted livery packages if conversion failed - @Benjozork
1. [UI] Rename #help to #support to adopt current Discord channel naming - @FoxtrotSierra6829, @davidwalschots
1. [LOGIC] Request the user to set 'Community' folder manually if it cannot be found, fixes error message - @FoxtrotSierra68
1. [ADDONS] Update "Experimental" version text for A32NX - @FoxtrotSierra6829
```

**Review by Benjozork — COMMENTED** [review]
can't approve as I am the author, but LGTM.

thanks for the hard work @FoxtrotSierra6829 !

**Pull request merged**
